### PR TITLE
tailscale: update to 1.36.0.

### PR DESCRIPTION
**Pull request by nilium**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)

### CODE REVIEW

File: srcpkgs/tailscale/template [hunk 1]
```diff
@@ -13,7 +13,7 @@ maintainer="Noel Cower <ncower@nil.dev>"
 license="BSD-3-Clause"
 homepage="https://tailscale.com"
 distfiles="https://github.com/tailscale/tailscale/archive/v${version}.tar.gz"
-checksum=@6b48993cee06397e22fd70c71d93ec9b2431cdb32a7e5cb37b515f7d34cbadb7
+checksum=@687833114a7294dc98c658502a4e4ea997bd467ad42c901edb4639935d71729d
```
Comment (paper42) [inline srcpkgs/tailscale/template:16]: Since this is a GitHub tarball, the hash should be stable, so @ shouldn't be necessary, why was it added?

Comment (nilium) [reply srcpkgs/tailscale/template:16]: Actually not sure -- came in via xgensum, so just going to remove it.

Comment (paper42) [reply srcpkgs/tailscale/template]: @ means it's a checksum of the tarball content instead of the tarball itself. It shouldn't be used almost anywhere.

Comment (nilium) [reply srcpkgs/tailscale/template]: I'm aware.

**Pull request merged**